packages/core-web/webpack.dev.js

```diff
@@ -47,6 +47,9 @@ module.exports = {
   serverEntry: (cb, rootFolder) => {
     const webpackServerDevConfig = merge(serverConfig, {
       mode: 'development',
+      output: {
+        path: path.join(rootFolder, 'dist'), // Change the output path based on the user defined root
```

Comment:
Lgtm 👍 I think this approach would be slightly less ambiguous too in case use of `process.cwd` changes in webpack.